js/src/pages/ProjectTeamPage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback, useEffect } from "react";
+import { Card, Col, Row } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
+import { useDispatch, useSelector } from "react-redux";
+import { useParams } from "react-router";
+
+import { TeamMembersList } from "../features/project/components/Team/TeamMembersList";
+import {
+  fetchProjectAsync,
+  fetchProjectTeamAsync,
+} from "../features/project/actions";
+import { getProject, getProjectTeam } from "../features/project/selectors";
+
+export const ProjectTeamPage: React.FC = () => {
+  const dispatch = useDispatch();
+  const { id, slug } = useParams<{ id: string; slug: string }>();
+
+  const fetchProjectTeam = useCallback(
+    (id: string) => {
+      dispatch(fetchProjectTeamAsync.request(id));
+    },
+    [dispatch],
+  );
+
+  const fetchProject = useCallback(
+    (slug: string) => {
+      dispatch(fetchProjectAsync.request(slug));
+    },
+    [dispatch],
+  );
+
+  const team = useSelector(getProjectTeam);
+  const project = useSelector(getProject);
+
+  useEffect(() => {
```

Comment:
I think we should separate this into 2 different hooks, because `fetchProjectTeam` and `fetchProject` are independent

Reply:
fix